.vscode/launch.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
can we .gitignore this? but also this is mine
```
    {
      "type": "lldb",
      "request": "launch",
      "name": "run agc",
      "program": "${workspaceFolder}/alan/avm/target/debug/alan",
      "args": ["run", "out1.agc"],
      "cwd": "${workspaceFolder}",
      "sourceLanguages": ["rust"],
      "preLaunchTask": "build avm"
    },
```

adding the source language made it the debugger more amicable